uk.co.compucorp.civicrm.contactactionsmenu/info.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<extension key="uk.co.compucorp.civicrm.contactactionsmenu" type="module">
```

Comment:
To keep it consistent with the other extension, I think it would be better to call it `uk.co.compucorp.civicrm.hrcontactactionsmenu`, as it will be a CiviHR extension for now. We can rename it If we want to make it a standalone extension in the future.